server/controllers/medical/patients/index.js

```diff
@@ -687,6 +688,15 @@ function getDebtorBalance(req, res, next) {
  * returns the stock Movements to the patient.
  */
 function getStockMovements(req, res, next) {
+  const uid = req.params.uuid;
+  stockMovementByPatient(uid)
+    .then((result) => {
+      res.status(200).json(result);
+    })
+    .catch(next);
+}
+
+function stockMovementByPatient(patientUuid) {
```

Comment:
We really need more information here - it would be nice to list every item of stock that was given to the patient, not just the reference of their movement.  So, we would want:
  1. The inventory item
  2. The stock movement document reference
  2. The quantity given
  3. The type (comprimes/boite/etc)
  4. The lot number
  5. The unit price of the stock
  6. The total price of the stock.

Reply:
We would probably then want a total at the bottom of this with the total value of stock distributed to the patient.